lib/thrift.ex

```diff
@@ -51,6 +51,58 @@ defmodule Thrift do
   on-demand. By default, it uses the same project configuration as the
   compiler task above, but options can also be specified using command line
   arguments.
+
+  ### Thrift Definitions
+
+  Given some Thrift type definitions:
+
+      namespace elixir Thrift.Test
+
+      exception UserNotFound {
+        1: string message
+      }
+
+      struct User {
+        1: i64 id,
+        2: string username,
+      }
+
+      service UserService {
+        bool ping(),
+        User get(1: i64 id) throws (1: UserNotFound e),
+        bool delete(1: i64 id),
+      }
```

Comment:
Maybe we should put this code in backticks so we can specify the language
````
```thrift
# thrift code
```
````
Hex docs use highlight.js for highlighting, which supports thrift, so in theory it will render nicely. In practice I'm not sure if it will work.

Reply:
It unfortunately doesn't, perhaps due to ex_doc's recent switch to [makeup](https://hex.pm/packages/makeup).

... but I'll change the Thrift examples to fenced code blocks because they look nicer unhighlighted than they do highlighted as Elixir code, and maybe we'll get Thrift highlighting support in the future.